appcues/src/main/java/com/appcues/ui/modal/DialogModal.kt

```diff
@@ -69,6 +95,8 @@ internal fun DialogModal(
 ) {
     val isDark = isSystemInDarkTheme()
     val density = LocalDensity.current
+    val viewModel = LocalViewModel.current
```

Comment:
it appears that due to our snapshot testing set up, I cannot currently access the composition local for `LocalViewModel` here 😬 need to look into options

Reply:
reworked this to use an abstraction layer for testing in https://github.com/appcues/appcues-android-sdk/pull/642/commits/f482f85b2a1490e277bf0540267f2267d54be745